admin/admin-filters-term.php

```diff
@@ -478,7 +478,9 @@ public function term_lang_choice() {
 					$args = array_merge( $args, array( 'link' => 'edit' ) );
 				}
 
-				if ( $tag_cloud = wp_tag_cloud( $args ) ) {
+				$tag_cloud = wp_tag_cloud( $args );
+
+				if ( ! empty( $tag_cloud ) && is_string( $tag_cloud ) ) {
```

Comment:
By default `format` argument is set to `flat` so if return isn't void, it must be a string.
PHPStan didn't catch it?

Reply:
I don't know what changed but the return type is simply `@return void|string|string[]` in the stubs, while it should be a conditional return type based on `format` and `echo`.

Reply:
Well, maybe some day phpstan-wordpress will support the conditional return type for `wp_tag_cloud()`.

Reply:
I guess that the dynamic return type of `wp_tag_cloud()` is not handled. If you are sure that we can get only a string then I suggest to document it as 
```php
/** @var string **/
tag_cloud = wp_tag_cloud( $args );
```
You can also write a PR for PHPStan WordPress.

Reply:
You're right, I'll do that.
The dynamic return type seems to be [a work in progress](https://github.com/szepeviktor/phpstan-wordpress/pull/147).

Reply:
Done in [4169e15](https://github.com/polylang/polylang/pull/1399/commits/4169e15958c28d37e2c95667b8f76fdc55d9f76d).